mypy/test/testsolve.py

```diff
@@ -2,10 +2,11 @@
 
 import typing
 
-from mypy.myunit import Suite, assert_equal
+from mypy.myunit.assertions import *
```

Comment:
Keep in mind that `mypy.myunit.assertions` is intended to gain *dozens* of `assert_foo` functions over time.

Asking developers to list them all manually will just make them stop writing tests.

***

There are three possibilities for sane assertions:

* Rewrite assertions like `py.test` (doable, but hard).
* Force all tests to occur within a class so the all the assertions will be available as members of `self`. (this *could* be done via a fixture argument, but that would be very counterintuitive and not extensible).
* Use glob imports.

To me, glob imports were the obvious answer. They are *generally* discouraged, but not forbidden because they are extremely useful in a very few cases. This is one of them, and it is safe because all exported functions from `mypy.myunit.assertions` start with `assert_` (except for `fail`).